locus/src/main/java/com/birjuvachhani/locus/Locus.kt

```diff
@@ -125,6 +128,7 @@ object Locus {
      */
     @Throws(IllegalStateException::class)
     fun startLocationUpdates(context: Context): LiveData<LocusResult> {
+        config.locationRequest = getLocusLocationRequest(context.getAvailableService())
```

Comment:
Am I understanding this wrong or this just overrides locationRequest with default settings?

Reply:
It will update the value of the locationRequest with the available service in this context.

Reply:
I get that. However, what happens to user's custom request? This uses `getHMSLocationRequest` and `getGMSLocationRequest` methods which always returns a new instance of the request. User's custom settings are gone. You're not supposed to do that!

<img width="641" alt="image" src="https://github.com/BirjuVachhani/locus-android/assets/20423471/b00c4809-e684-4565-9c40-afe93b1fdd32">

Reply:
These are the logs I am getting. It is clear that your code here is resetting user's location request configuration.

Please make sure this never happens!

<img width="726" alt="image" src="https://github.com/BirjuVachhani/locus-android/assets/20423471/9770b7f1-1088-425b-a2c2-21cef5d2d8a2">
<img width="996" alt="image" src="https://github.com/BirjuVachhani/locus-android/assets/20423471/5c092171-9118-4cb7-b4dc-bd829b45b3f7">
<img width="974" alt="image" src="https://github.com/BirjuVachhani/locus-android/assets/20423471/a02ab6b4-a427-4c26-ad61-3e41da60ebd0">

Reply:
Thanks for the details.

Reply:
Hello @BirjuVachhani,
I hope you are doing well.
I have made some changes regarding the user configurations on the Location Request please check it [c7a06fb](https://github.com/BirjuVachhani/locus-android/pull/87/commits/c7a06fb5c757bc4253b4e3f73629d768b35ff41e).